GAMES.json

```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260": true,
```

Comment:
yea my bad, true should be gone too.

Take a look at examples in readme: https://github.com/SteamDatabase/SteamLinux#contribution